### PR TITLE
Add rosaction CLI tool

### LINKED ISSUE #44: Feature to get a list of running action servers
[Asked here](http://answers.ros.org/question/222748/list-action-servers/). While the proposed way that uses `rostopic list` (thus is a bit of a hack) seems to work perfectly, I like to have a feature to do that. I wouldn't mind opening a PR but just wonder:
- What form is the best? A command like `rosaction` with a `list` verb may sound too much, since there may not be many other things to do with `rosaction` command.
- Where's the right place? This should be just determined once "what form" is resolved.

### PR DESCRIPTION
**Pull request by maximkulkin**
Added rosaction CLI tool to list/get info/send actions, similar to rosservice tool.

Supported commands:
* `rosaction list`
* `rosaction info /action`
* `rosaction type /action`
* `rosaction type /action goal`
* `rosaction type /action feedback`
* `rosaction type /action result`
* `rosaction send /action "goal: 123"`

Closes #44

### CODE REVIEW

**Comment by maximkulkin** [comment]
Guys this PR is ready and passing. I'm a bit concerned on the whole "wait" thing in tests: you basically send something and wait for response and that waiting has a timeout but if system is under load, it might not be quick to respond and the test will fail. Although this approach seems to be ubiquitous. Otherwise please advise.

**Comment by maximkulkin** [comment]
This PR is ready to be merged.

**Comment by maximkulkin** [comment]
Ping! Can we get this PR merged?

File: CMakeLists.txt [hunk 1]
```diff
@@ -23,7 +23,9 @@ add_library(actionlib src/connection_monitor.cpp src/goal_id_generator.cpp)
 target_link_libraries(actionlib ${catkin_LIBRARIES} ${Boost_LIBRARIES})
 add_dependencies(actionlib actionlib_gencpp)
 
-catkin_install_python(PROGRAMS tools/axclient.py tools/axserver.py tools/dynamic_action.py tools/library.py
+catkin_add_env_hooks(15.rosaction SHELLS bash DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/env-hooks)
+
+catkin_install_python(PROGRAMS tools/rosaction tools/axclient.py tools/axserver.py tools/dynamic_action.py tools/library.py
```
Comment (mjcarroll) [inline CMakeLists.txt:28]: You are installing some files that don't seem to be part of this PR.  Were these intended to be here?

File: src/actionlib/utils.py [hunk 2]
```diff
@@ -0,0 +1,140 @@
+import rosgraph
```
Comment (mjcarroll) [inline src/actionlib/utils.py:1]: I believe that this is conceptually similar enough to the McGill code that it warrants including the full MIT license header as well as their copyright statement and a link to the original repository.

**Comment by felixvd** [comment]
Just a +1 to say that if this works as expected it's a great addition that most users probably simply expect to be there (like I did, once).

**Comment by fujitatomoya** [comment]
+1, i believe that conceptually action should have own interface to user.

**Review by mjcarroll — CHANGES_REQUESTED** [review]
Sorry, I started this review an never submitted it.

File: CMakeLists.txt (hunk 1 above)
Comment (mmoerdijk) [reply CMakeLists.txt:28]: I don't think he is, as hey only added the `tools/rosaction` and the env hooks for it.

File: src/actionlib/utils.py (hunk 2 above)
Comment (mmoerdijk) [reply src/actionlib/utils.py:1]: I agree @maximkulkin could you dust off this PR and add the [license of mcgill](https://github.com/mcgill-robotics/rosaction/blob/master/LICENSE) to the files?

**Pull request closed without merging**